doc/source/tune/api/logging.rst

```diff
@@ -59,8 +59,10 @@ See the :doc:`tutorial here </tune/examples/tune-mlflow>`.
 
 .. autosummary::
     :nosignatures:
+    :toctree: doc/
 
-    tune.logger.mlflow.MLflowLoggerCallback
+    ~air.integrations.mlflow.MLflowLoggerCallback
```

Comment:
qq: why the `~`, is it for correctness or for look & feel

Reply:
Not for correctness, only look and feel - I can remove this.